kloppy/tests/files/epts_metrica.xml

```diff
@@ -0,0 +1,306 @@
+<?xml version="1.1" encoding="UTF-8"?>
+<tns:FIFADataTransferFormatEPTS
+	xmlns:tns="FIFADataTransferFormatEPTSNamespace/FIFADataTransferFormatEPTS"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Please delete as possible from the xml to let the test still pass. This makes it easier find problems when the test breaks.

Reply:
I think there is something missing. Did you want to say as much as possible?

Reply:
Ah yes. As much as possible 👍